packages/zwave-js/src/lib/controller/Controller.ts

```diff
@@ -1928,19 +2004,32 @@ export class ZWaveController
 			return false;
 		}
 
+		// BUGBUG: fix me, same logic as beginInclusion, probably an incoming option...
+		// if (options.isLongRange && !this.isLongRange()) {
+		// 	throw new ZWaveError(
+		// 		`Invalid long range inclusion on a non-LR host`,
+		// 		ZWaveErrorCodes.Argument_Invalid,
+		// 	)
+		// }
+		const isLongRange = this.isLongRange();
+
```

Comment:
Again, all this "include as long range or not" logic needs overhauled once we have a plan for how the UI/clients will interact with the driver about it.